lib/v2/nua/utils.js

```diff
@@ -13,45 +26,45 @@ async function ProcessList(newsUrl, baseUrl, listName, listDate, webPageName) {
 
     const pageName = $(webPageName).text();
 
-    let artiLink;
-
     const items = $(listName)
         .toArray()
         .map((item) => {
             item = $(item);
-            if (item.find('a').attr('href').includes('http')) {
-                artiLink = item.find('a').attr('href');
-            } else {
-                artiLink = baseUrl + item.find('a').attr('href');
-            }
+            const href = $(item).find('a').attr('href');
```

Comment:
since you have `item = $(item)` above, you can use
```suggestion
            const href = item.find('a').attr('href');
```